command/intake.py

```diff
@@ -0,0 +1,50 @@
+import config
+import wpilib
+from robotpy_toolkit_7407 import SubsystemCommand
+from oi.keymap import Controllers
+from subsystem import Intake
+from robotpy_toolkit_7407.utils.units import radians
+
+
+class SetIntake(SubsystemCommand[Intake]):
+    def __init__(self, subsystem: Intake, intake_active: bool, go_cube: bool, go_cone: bool):
```

Comment:
Instead of `go_cube` and `go_cone`, use a dictionary like 
`game_piece`
and in constants:
`game_piece = {
 'cone':0
 'cube':1
}`
then use this inside the command